src/server/api/routers/set.ts

```
@@ -66,6 +70,164 @@ export const setRouter = createTRPCRouter({
+    .input(getInfiniteSetsSchema)
+    .query(async ({ ctx, input }) => {
+      console.log(
+        `🤖 - [set/getInfinite] - attempting to query sets for organization ${input.organizationId}`,
+        { queryInput: input },
+      );
+
+      const limit = input.limit ?? 10;
+
+      let validatedEventTypeId: string | undefined;
+      if (!!input.eventTypeId) {
+        const eventType = await ctx.db.query.eventTypes.findFirst({
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)